### DISCUSSION TITLE
Keeping sound settings together

### DISCUSSION THREAD

**Discussion by Gene703122** [nvaccess/nvda#14064] [Ideas]
### Is your feature request related to a problem? Please describe.
Many settings regarding sounds are in the general category.  But the setting to turn the sound off or on when suggestions appear, as in the start menu is in object presentation settings.  I haven't checked to see if any other sound settings are in other places.
### Describe the solution you'd like
All sound settings should be in one place, in general settings.
### Describe alternatives you've considered
None.
### Additional context
While technically it may make sense to have the sound setting being discussed in object presentation settings, as a matter of function it makes none.

**Comment by josephsl** [comment]
Hi,

I disagree for two reasons:

1. General settings apply to all profiles, whereas object presentation settings can be customized on a per-profile basis.
2. Suggestion sounds are not really sounds coming from NVDA - rather, they are in response to auto-suggest accessibility events coming from various objects such as Windows 10/11 Start menu, Notepad++, and others yet to be discovered.

While it makes sense to keep a specific category of settings together, it is also important to see why settings were added in specific places, and suggestions sound is one of these that cannot be combined with other sound settings. We might as well move spelling error sound setting from keyboard settings to general settings, but that also runs into the problems outlined above (in this case, spelling error settings is tied to typing, thus it is best to keep it in keyboard settings panel).

Thanks.
